docs/schema.md

```diff
@@ -319,6 +319,17 @@ The defined database prefixes and their "home" databases are:
         </ul>
       </td>
     </tr>
+    <tr>
+      <td><code>RSEC</code></td>
+      <td><a href="https://github.com/RConsortium/r-advisory-database">RConsortium Advisory Database</a></td>
+      <td>
+        <ul>
+          <li>How to contribute: <a href="https://github.com/RConsortium/r-advisory-database#readme">https://github.com/RConsortium/r-advisory-database#readme</a></li>
+          <li>Source URL: <code>https://osv.dev/vulnerability/&lt;ID&gt;</code></li>
+          <li>OSV Formatted URL: <code>https://github.com/RConsoritum/r-advisory-database/&lt;package&gt;/&lt;ID&gt;.yaml</code> (unofficial)</li>
```

Comment:
There is a typo in the URL

```suggestion
          <li>OSV Formatted URL: <code>https://github.com/RConsortium/r-advisory-database/&lt;package&gt;/&lt;ID&gt;.yaml</code> (unofficial)</li>
```

Reply:
Also, does the constructed URL work though? E.g.

https://github.com/RConsortium/r-advisory-database/readxl/RSEC-2023-0.yaml

would not lead to the advisory if I'm not mistaken.

Reply:
This should work directly:

```suggestion
          <li>OSV Formatted URL: <code>https://raw.githubusercontent.com/RConsortium/r-advisory-database/main/vulns/&lt;package&gt;/&lt;ID&gt;.yaml</code></li>
```

Reply:
I've fixed the link here.